usr/lib/pkcs11/common/asn1.c

```diff
@@ -28,2007 +28,2012 @@
 
 //
```

Comment:
I don't like these kind of empty comments, they are somewhat useless... 
I guess its hard to catch them with the indent tool....